cli/config/config.go

```diff
@@ -100,29 +106,36 @@ func load(configDir string) (*configfile.ConfigFile, error) {
 	file, err := os.Open(filename)
 	if err != nil {
 		if os.IsNotExist(err) {
-			//
-			// if file is there but we can't stat it for any reason other
-			// than it doesn't exist then stop
+			// It is OK for no configuration file to be present, in which
+			// case we return a default struct.
 			return configFile, nil
 		}
-		// if file is there but we can't stat it for any reason other
-		// than it doesn't exist then stop
-		return configFile, nil
+		// Any other error happening when failing to read the file must be returned.
+		return configFile, errors.Wrap(err, "loading config file")
```

Comment:
nit: maybe `errors.Wrapf(err, "loading config file: %s: ", filename)` like below so the errors have a consistent format?

Reply:
O left that out, because `os.Open` returns a  `os.PathError` / `fs.PathError`; https://pkg.go.dev/os#PathError and those already include the path, which would be `open /no/such/file: no such file or directory`, so manually adding the filename would now have the filename in the error twice.